### PR TITLE
Changes for Zammad 5

### PR DESCRIPTION
**Pull request by mgruner**
Just as a heads up: this change will be needed for Zammad 5.

https://github.com/zammad/zammad-docker-compose/issues/220

Changes:
* update memchached image
* changed memcached config
* added redis as websocket session store
* removed links in compose (should be superfluous)
* added cache fix
* changed filldb command
* updated ruby image
* addded nodejs build dependency
* added update note
* removed some whitepaces

### CODE REVIEW

**Comment by monotek** [comment]
Thanks for the hint!
Is there already an eta for 5.0?

**Comment by mgruner** [comment]
@thorsteneckel can you say something about ETA of Zammad 5?

**Comment by MrGeneration** [comment]
We're never announcing ETAs for releasing in public for very good reasons.
Please invole André in private.

**Comment by mgruner** [comment]
@monotek FYI, I also added the Ruby dependency change now that will be required for Zammad 5.

**Comment by monotek** [comment]
Thanks. I think i will use this to update the env file and so on too, as soon as teh first 5.0 stable image is available :)

**Comment by waja** [comment]
Ping

**Review by monotek — APPROVED** [review]

**Review by monotek — DISMISSED** [review]

File: containers/zammad/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN chmod +x /tmp/setup.sh; \
     /tmp/setup.sh builder
 
 
-FROM ruby:2.6.8-slim
+FROM ruby:2.7.3-slim
```
Comment (monotek) [inline containers/zammad/Dockerfile:27]:
```suggestion
FROM ruby:2.7.4-slim
```

**Comment by monotek** [comment]
Thanks! :)
Hopefully Github action will work soon enough again :D
https://www.githubstatus.com/

**Review by monotek — APPROVED** [review]

**Pull request merged**